packages/thirdweb/src/bridge/Sell.ts

```diff
@@ -0,0 +1,263 @@
+ *
+ * @example
+ * ```typescript
+ * import { Bridge, NATIVE_TOKEN_ADDRESS } from "thirdweb";
+ *
+ * const quote = await Bridge.Sell.prepare({
+ *   originChainId: 1,
+ *   originTokenAddress: NATIVE_TOKEN_ADDRESS,
+ *   destinationChainId: 10,
+ *   destinationTokenAddress: NATIVE_TOKEN_ADDRESS,
+ *   sellAmountWei: toWei("0.01"),
+ *   client: thirdwebClient,
+ * });
+ * ```
+ *
+ * This will return a quote that might look like:
+ * ```typescript
+ * {
+ *   originAmount: 1000000000000000000n,
+ *   destinationAmount:  9980000000000000000n,
+ *   blockNumber: 22026509n,
+ *   timestamp: 1741730936680,
+ *   estimatedExecutionTimeMs: 1000
+ *   transactions: [
+ *     {
+ *       to: NATIVE_TOKEN_ADDRESS,
+ *       value: 9980000000000000000n,
+ *       data: "0x",
+ *       chainId: 10,
+ *       type: "eip1559"
+ *     }
+ *   ],
+ *   expiration: 1741730936680,
+ *   intent: {
+ *     originChainId: 1,
+ *     originTokenAddress: NATIVE_TOKEN_ADDRESS,
+ *     destinationChainId: 10,
+ *     destinationTokenAddress: NATIVE_TOKEN_ADDRESS,
+ *     sellAmountWei: 1000000000000000000n
+ *   }
+ * }
+ * ```
+ *
+ * ## Sending the transactions
+ * The `transactions` array is a series of [ox](https://oxlib.sh) EIP-1559 transactions that must be executed one after the other in order to fulfill the complete route. There are a few things to keep in mind when executing these transactions:
+ *  - Approvals and other preparation transactions are not included in the transactions array.
+ *  - All transactions are assumed to be executed by the `sender` address, regardless of which chain they are on. The final transaction will use the `receiver` as the recipient address.
+ *  - If an `expiration` timestamp is provided, all transactions must be executed before that time to guarantee successful execution at the specified price.
+ *
+ * NOTE: To get the status of each transaction, use `Bridge.status` rather than checking for transaction inclusion. This function will ensure full bridge completion on the destination chain.
+ *
+ * You can access this functions input and output types with `Sell.prepare.Options` and `Sell.prepare.Result`, respectively.
+ *
+ * @param options - The options for the quote.
+ * @param options.originChainId - The chain ID of the origin token.
+ * @param options.originTokenAddress - The address of the origin token.
+ * @param options.destinationChainId - The chain ID of the destination token.
+ * @param options.destinationTokenAddress - The address of the destination token.
+ * @param options.sellAmountWei - The amount of the origin token to sell.
+ * @param options.sender - The address of the sender.
+ * @param options.receiver - The address of the recipient.
+ * @param options.client - Your thirdweb client.
+ *
+ * @returns A promise that resolves to a non-finalized quote for the requested buy.
+ *
+ * @throws Will throw an error if there is an issue fetching the quote.
+ */
```

Comment:
should we mark these APIs as @beta in case we iterate a bit on the types?

Reply:
Yes for sure